flake8_vedro/rules/VDR313.md

```diff
@@ -0,0 +1,25 @@
+# VDR313. contexts should have an assertion
+The Vedro framework Сontexts should contain assertion:)
```

Comment:
1. "The Vedro framework Сontexts should contain an assertion"
2. It would be great to explain why this rule exists here (something about ensuring changes of application)